### PR TITLE
add easyconfigs for NCO/ANTLR with intel/2014b toolchain

### PR DESCRIPTION
**Pull request by boegel**
based on #1169 by @pm67nz

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/6648047daefd8cf668d1 for a full test report.

File: easybuild/easyconfigs/a/ANTLR/ANTLR-2.7.7-intel-2014b.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+name = 'ANTLR'
+version = '2.7.7'
+
+homepage = "http://www.antlr2.org"
+description = """ANother Tool for Language Recognition"""
+
+toolchain = {'name': 'intel', 'version': '2014b'}
+
+sources = ['antlr-%(version)s.tar.gz']
```
Comment (wpoely86) [inline easybuild/easyconfigs/a/ANTLR/ANTLR-2.7.7-intel-2014b.eb]: `SOURCELOWER_TAR_GZ`?

**Comment by wpoely86** [comment]
lgtm, ready to merge

**Comment by boegel** [comment]
Going in, thanks for the review @wpoely86!

**Pull request merged**